examples/workload-multiple-secrets/main.go

```diff
@@ -47,6 +44,35 @@ func main() {
 		return
 	}
 
-	// d.Data is a collection of Secrets.
-	fmt.Println(d.Data)
+	// Check if d.Data is a JSON array
```

Comment:
This change was to demo a use case that if you want you can send a larger secret in chunks, and then the workload can parse those chunks and combine them together.

That is useful if the secret